packages/monorepo-split-github-action/entrypoint.sh

```diff
@@ -50,6 +51,14 @@ note "Cleaning destination repository of old files"
 find "$CLONE_DIR" | grep -v "^$CLONE_DIR/\.git" | grep -v "^$CLONE_DIR$" | xargs rm -rf # delete all files (to handle deletions)
 ls -la "$CLONE_DIR"
 
+if test ! -z "$COMMIT_MESSAGE"
+then
+    ORIGIN_COMMIT="https://github.com/$GITHUB_REPOSITORY/commit/$GITHUB_SHA"
+    COMMIT_MESSAGE="${COMMIT_MESSAGE/ORIGIN_COMMIT/$ORIGIN_COMMIT}"
+else
+    COMMIT_MESSAGE=$(git show -s --format=%B "$GITHUB_SHA")
```

Comment:
@TomasVotruba What do you think? I'm using this trick for [LongRunning](https://github.com/LongRunning/LongRunning). 

See mono commit https://github.com/LongRunning/LongRunning/commit/62eae751184e87506870c096e76084b53cbb0fdb
and split commit https://github.com/LongRunning/core/commit/a8b565172fccda4cbef1dbaf52eda2fbfa0dd07a

Reply:
Cool, let's try it